downloads.html

```diff
@@ -15,6 +15,14 @@
   <div class="container content">
 
     <h1>Downloads</h1>
+    <br>
```

Comment:
You shouldn't use a `<br>` here. If you want more space above the <p>, you can use `<p class="mt-5">`